source/standalone/tutorials/04_sensors/benchmark_cameras.py

```diff
@@ -0,0 +1,668 @@
+    cfg.func("/World/ground", cfg)
+    # -- Lights
+    cfg = sim_utils.DistantLightCfg(intensity=3000.0, color=(0.75, 0.75, 0.75))
+    cfg.func("/World/Light", cfg)
+
+    # Create a dictionary for the scene entities
+    scene_entities = {}
+
+    # Xform to hold objects
+    prim_utils.create_prim("/World/Objects", "Xform")
+    # Random objects
+    for i in range(num_objects):
+        # sample random position
+        position = np.random.rand(3) - np.asarray([0.05, 0.05, -1.0])
+        position *= np.asarray([1.5, 1.5, 0.5])
+        # sample random color
+        color = (random.random(), random.random(), random.random())
+        # choose random prim type
+        prim_type = random.choice(["Cube", "Cone", "Cylinder"])
+        common_properties = {
+            "rigid_props": sim_utils.RigidBodyPropertiesCfg(),
+            "mass_props": sim_utils.MassPropertiesCfg(mass=5.0),
+            "collision_props": sim_utils.CollisionPropertiesCfg(),
+            "visual_material": sim_utils.PreviewSurfaceCfg(diffuse_color=color, metallic=0.5),
+            "semantic_tags": [("class", prim_type)],
+        }
+        if prim_type == "Cube":
+            shape_cfg = sim_utils.CuboidCfg(size=(0.25, 0.25, 0.25), **common_properties)
+        elif prim_type == "Cone":
+            shape_cfg = sim_utils.ConeCfg(radius=0.1, height=0.25, **common_properties)
+        elif prim_type == "Cylinder":
+            shape_cfg = sim_utils.CylinderCfg(radius=0.25, height=0.25, **common_properties)
+        # Rigid Object
+        obj_cfg = RigidObjectCfg(
+            prim_path=f"/World/Objects/Obj_{i:02d}",
+            spawn=shape_cfg,
+            init_state=RigidObjectCfg.InitialStateCfg(pos=position),
+        )
+        scene_entities[f"rigid_object{i}"] = RigidObject(cfg=obj_cfg)
+
+    # Sensors
+    standard_camera = create_cameras(
+        num_cams=num_standard_cams, data_types=standard_camera_replicators, height=height, width=width
+    )
+    tiled_camera = create_tiled_cameras(
+        num_cams=num_tiled_cams, data_types=tiled_camera_replicators, height=height, width=width
+    )
+    ray_caster_camera = create_ray_caster_cameras(
+        num_cams=num_ray_caster_cams,
+        data_types=ray_caster_camera_replicators,
+        mesh_prim_paths=mesh_prim_paths,
+        height=height,
+        width=width,
+    )
+    # return the scene information
+    if tiled_camera is not None:
+        scene_entities["tiled_camera"] = tiled_camera
+    if standard_camera is not None:
+        scene_entities["standard_camera"] = standard_camera
+    if ray_caster_camera is not None:
+        scene_entities["ray_caster_camera"] = ray_caster_camera
+    return scene_entities
+
+
+def numpy_to_pcd(xyz: np.ndarray) -> o3d.geometry.PointCloud:
+    """Convert a NumPy Nx3 pointcloud to an Open3d Nx3 pointcloud for easy plotting"""
+    pcd = o3d.geometry.PointCloud()
+    pcd.points = o3d.utility.Vector3dVector(xyz)
+    return pcd
+
+
+def run_simulator(
```

Comment:
Please reorder the functions in the right order (i,e, the order in which they are called). Makes it easier to read the code later.

```python
"""
Helper functions.
"""

"""
Simulation
"""
```